### DISCUSSION TITLE
Ei glr estou com uma duvida, eu consigo usar um codigo html e css no filament? Coloquei meu codigo html no filament e não consigo apontar para o css>

### DISCUSSION THREAD

**Discussion by vinix13x** [filamentphp/filament#11851] [Help]
### Package

Panel builder

### Package Version

3

### How can we help you?

Help?

**Comment by leandrocfe** [comment]
Quando postar uma questão, tente fornecer mais detalhes, compartilha o código que está usando, etc.
Se você procura utilizar css dentro do Filament, você pode criar um tema personalizado: https://filamentphp.com/docs/3.x/panels/themes#creating-a-custom-theme
